frontend/packages/metal3-plugin/locales/zh/metal3-plugin.json

```diff
@@ -0,0 +1,202 @@
+  "all workloads have already been moved, but {{length}} daemon sets have been skipped.": "所有工作负载都已移动，但已经跳过 {{length}} 守护进程集。",
+  "all workloads have already been moved, but {{podListlength}} static pods and {{daemonListlength}} daemon sets have been skipped.": "所有工作负载都已移动，但已跳过 {{podListlength}} 静态 pod 和 {{daemonListlength}} 守护进程集。",
+  "The host will be powered off gracefully.": "主机将安全关闭。",
+  "To power off gracefully, <1>start maintenance</1> on this host to move all managed workloads to other nodes in the cluster.": "要关闭电源，在这个主机上<1>启动维护</1>，将所有受管工作负载移动到集群中的其他节点。",
+  "Workloads will not be moved before the host powers off.": "在主机关闭前不会移动工作负载。",
+  "The host will power off immediately as if it were unplugged.": "主机将立即关闭，就像它被断电。",
+  "Power off immediately": "立即关机",
+  "Applications may be temporarily disrupted.": "应用程序可能会临时被影响。",
+  "Workloads currently running on this host will not be moved before powering off. This may cause service disruptions.": "在关机前，在这个主机上当前运行的工作负载不会被移动。这可能会导致服务中断。",
+  "Power Off Host": "关闭主机",
+  "Host is ready to be gracefully powered off. The host is currently under maintenance and {{message}}": "主机已准备好安全关闭。主机当前处于维护状态且 {{message}}",
+  "Host is ready to be gracefully powered off.": "主机已准备好被安全关机。",
+  "Failed to load data.": "加载数据失败。",
+  "Failed to load subresources.": "加载子资源失败。",
+  "The bare metal host is not healthy.": "裸机主机不健康。",
+  "The host cannot be powered off gracefully untils its health is restored.": "在恢复主机健康前，不能安全关闭它。",
+  "The node is starting maintenance.": "该节点正在开始维护。",
+  "The node cannot be powered off gracefully until it finishes entering maintenance.": "在完成对节点进行维护操作前，不能安全关闭该节点。",
+  "The node is stopping maintenance.": "节点停止维护。",
+  "The node cannot be powered off gracefully while it is exiting maintenance.": "在退出维护时无法安全关闭该节点。",
+  "This node contains DaemonSet pods.": "此节点包含 DaemonSet pod。",
+  "These DaemonSets will prevent some pods from being moved. This should not prevent the host from powering off gracefully.": "这些 DaemonSet 会阻止一些 pod 被移动。这不应该阻止主机安全地关闭。",
+  "This host contains unmanaged static pods.": "此主机包含非受管静态 Pod。",
+  "These pods must be moved manually to continue running after the host powers off.": "这些 pod 必须手动移动，才能在主机关闭后继续运行。",
+  "The bare metal host {{name}} will be restarted gracefully after all managed workloads are moved.": "在所有受管工作负载 被移动后，裸机主机 {{name}} 会被安全重启。",
+  "The bare metal host {{name}} will be restarted gracefully.": "裸机主机 {{name}} 将会安全地重启。",
+  "Restart Bare Metal Host": "重启裸机主机",
+  "All managed workloads will be moved off of this node. New workloads and data will not be added to this node until maintenance is stopped.": "所有受管工作负载都将移出这个节点。在停止维护前，不会将新工作负载和数据添加到这个节点。",
+  "If the node does not exit maintenance within <1>30 minutes</1>, the cluster will automatically rebuild the node&apos;s data using replicated copies": "如果节点在 <1>30 分钟内</1>没有退出维护，集群将自动使用复制副本重建节点的数据",
```

Comment:
should `&apos;` be `'`

Reply:
I can't update these file keys without touching the metal3-plugin code. I can file a separate bug for these.

Reply:
See https://bugzilla.redhat.com/show_bug.cgi?id=1908434.

Reply:
PR merged; this is all set now.